eventmesh-storage-plugin/eventmesh-storage-pulsar/src/main/java/org/apache/eventmesh/storage/pulsar/consumer/PulsarConsumerImpl.java

```diff
@@ -122,8 +124,19 @@ public void commit(EventMeshAction action) {
 
         String consumerKey = topic + PulsarConstant.KEY_SEPARATOR + properties.getProperty(Constants.CONSUMER_GROUP)
             + PulsarConstant.KEY_SEPARATOR + properties.getProperty(Constants.CLIENT_ADDRESS);
+        
+        String dlqTopic = subTopic + "-DLQ";
+        
+        String retryTopic = subTopic + "-RETRY";
+        
         org.apache.pulsar.client.api.Consumer<byte[]> consumer = pulsarClient.newConsumer()
             .topic(subTopic)
+            .enableRetry(true)
+            .deadLetterPolicy(DeadLetterPolicy.builder()
+                    .deadLetterTopic(dlqTopic)
+                    .retryLetterTopic(retryTopic)
```

Comment:
You have defined the retry topic here, why do you still need to manually send the message to the retry topic in `PulsarRetryStrategyImpl`? Doesn't Pulsar automatically send failed messages to the retry topic?

Reply:
Fixed.